src/ext/oblivc/obliv_bits.h

```diff
@@ -40,13 +43,17 @@ void __obliv_c__setBitOr(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitXor(OblivBit* dest,const OblivBit* a,const OblivBit* b);
 void __obliv_c__setBitNot(OblivBit* dest,const OblivBit* a);
 void __obliv_c__flipBit(OblivBit* dest); // Avoids a struct copy
+void __obliv_c__floatAdd(OblivBit* dest,const OblivBit* a,const OblivBit* b);
```

Comment:
The reason that the naming convention is different is because the floatAdd function represents an arithmetic circuit-based operation whereas the other functions are just individual bitwise gates.